nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/hosted/HelmHostedRecipe.groovy

```diff
@@ -89,6 +89,7 @@ class HelmHostedRecipe
 
     [indexMatcher(), packageMatcher()].each { matcher ->
       builder.route(new Route.Builder().matcher(matcher)
+          .handler(formatHighAvailabilitySupportHandler)
```

Comment:
move HA handler after security handler (in all routes)

Reply:
fixed, please take a look 
https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/72/commits/ce51f279a8bb6093fc738d5dc0bb650caf6be079